app/helpers/application_helper.rb

```diff
@@ -1,2 +1,28 @@
 module ApplicationHelper
+  def aws_config
+    aws_secret_file = Rails.configuration.service['aws_secret_file']
```

Comment:
it should be usable with IAM roles as well. And move setup of aws to config/initializers/